app/src/main/java/com/infomaniak/mail/data/models/thread/Thread.kt

```diff
@@ -203,17 +200,6 @@ class Thread : RealmObject {
         }
     }
 
-    private fun Date.formatNumericalDayMonthYear(): String {
-        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            // Adapts order of day, month and year according to the locale
-            val shortDateFormatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT)
-            toInstant().atZone(ZoneId.systemDefault()).toLocalDate().format(shortDateFormatter)
-        } else {
-            // Fallback on unambiguous date format for any locale
-            format(FORMAT_DATE_CLEAR_MONTH_DAY_ONE_CHAR)
-        }
-    }
-
```

Comment:
Do not remove it from there, only duplicated what you need, this will help avoid useless merge conflicts